com.avaloq.tools.ddk.xtext.ui.test/src/com/avaloq/tools/ddk/xtext/ui/templates/SimpleEnumTemplateVariableResolverTest.java

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Avaloq Evolution AG and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Avaloq Evolution AG - initial API and implementation
+ *******************************************************************************/
+
+package com.avaloq.tools.ddk.xtext.ui.templates;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.mockito.Mockito.mock;
+
+import java.util.List;
+
+import org.eclipse.jface.text.templates.TemplateException;
+import org.eclipse.jface.text.templates.TemplateVariable;
+import org.eclipse.xtext.XtextRuntimeModule;
+import org.eclipse.xtext.ui.editor.templates.XtextTemplateContext;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.google.inject.Guice;
+
+
+public class SimpleEnumTemplateVariableResolverTest {
+
+  private static XtextTemplateContext mockContext;
+  private static TemplateVariableResolverTestHelper helper;
+  private static SimpleEnumTemplateVariableResolver resolver;
+
+  @BeforeClass
+  public static void beforeClass() {
+    mockContext = mock(XtextTemplateContext.class);
+    helper = Guice.createInjector(new XtextRuntimeModule()).getInstance(TemplateVariableResolverTestHelper.class);
+    resolver = new SimpleEnumTemplateVariableResolver();
+  }
+
+  @AfterClass
+  public static void afterClass() {
+    mockContext = null;
+    helper = null;
+    resolver = null;
+  }
+
+  @Test(expected = NullPointerException.class)
+  public void testResolveValuesWithNullVariable() {
+    resolver.resolveValues(null, mockContext);
+  }
+
+  public void testResolveValuesWithNoParams() throws TemplateException {

```

Comment:
isn't the Test annotation missing here and in the next methods?

Reply:
Good point. Fixed.